components/backline/habitat/plan.sh

```diff
@@ -14,6 +13,19 @@ pkg_deps=(core/hab-plan-build
           core/vim
           core/ncurses)
 
+pkg_version() {
+  if [[ "$DO_FAKE_RELEASE" == "" ]]; then 
```

Comment:
To test of the existence of a variable,
```suggestion
  if [[ -z "${DO_FAKE_RELEASE:-}" ]]; then 
```
is the more common idiom, I believe

Though in this case, I'd reverse it so we can talk about positive conditions (which I find simpler to reason about):
```bash
  if [[ -n ${BUILDKITE_BUILD_ID:-}" ]]; then
    cat "$SRC_PATH/../../VERSION_FAKE"
  else
    cat "$SRC_PATH/../../VERSION"
  fi
```
You can read `if [[ -n ${foo:-}" ]]; then` as "if `foo` is set to something

Reply:
I actually did that with the powershell stuff - I'll switch the logic on the linux based plans.